doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "NASYS LoRaWAN® Outdoor Gateway"
```

Comment:
I see that the Official Docs uses `®` to describe LoRaWAN.  In the context of these docs, I think we can drop this.

(PS: The Docs also say "LoRaWAN® is a registered trademark of Semtech Corporation." but I think it's a trademark of the LoRa alliance and not Semtech 🤷‍♂ ).

Reply:
We indeed don't need to repeat the ® everywhere, but I don't know the exact rules. @johanstokking ?

Reply:
No we don't need to do that

Reply:
Removed it